doc/user-guide.adoc

```diff
@@ -37,9 +37,10 @@ spec:
     type: ClusterIP
```

Comment:
The user-guide.adoc in master branch is used by current users of beta operator releases, but these 1.0 doc changes wouldn't apply to their version. Please clone the content to user-guide-v1.adoc and add these there. We should also remove the previous set of doc changes from user-guide.adoc (i.e. startupProbe)